model_zoo/census_model_sqlflow/wide_and_deep/decorators.py

```diff
@@ -0,0 +1,12 @@
+def model_input_name(input_names):
```

Comment:
Can we place the decorator into a common util?

Reply:
Yes, I think we can put it into a separate python package.
The ML specialist want to commit the model into SQLFlow model zoo. He need add this decorator.

Reply:
We can move it into elasticdl_preprocessing at first.